Website/DesktopModules/Hotcakes/Core/Admin/Parts/PaymentMethods/PaypalExpress/App_LocalResources/Edit.ascx.resx

```diff
@@ -150,15 +148,7 @@
   <data name="ProductionMode.Text" xml:space="preserve">
     <value>Production Mode</value>
   </data>
-  <data name="rfvPassword.Text" xml:space="preserve">
```

Comment:
Is there no replacement validation for the new fields?

Reply:
Please let me know if it is well know.